src/customizations/volto/components/manage/Contents/ContentsItem.jsx

```diff
@@ -0,0 +1,409 @@
+              />
+            </Button>
+          )}
+        </Table.Cell>
+        <Table.Cell className={cx('', { 'dragging-cell': isDragging })}>
+          <Link
+            className="icon-align-name"
+            to={`${item['@id']}${item.is_folderish ? '/contents' : ''}`}
+          >
+            <div className="expire-align">
+              <Icon
+                name={getContentIcon(item['@type'], item.is_folderish)}
+                size="20px"
+                className="icon-margin"
+                color="#878f93"
+                title={item['Type'] || item['@type']}
+              />{' '}
+              <span title={item.title}> {item.title}</span>
+            </div>
+            {item.ExpirationDate !== 'None' &&
+              new Date(item.ExpirationDate).getTime() <
+                new Date().getTime() && (
+                <Button className="button-margin" size="mini">
+                  <FormattedMessage id="Expired" defaultMessage="Expired" />
+                </Button>
+              )}
+            {item.EffectiveDate !== 'None' &&
+              new Date(item.EffectiveDate).getTime() > new Date().getTime() && (
+                <Button className="button-margin effective-future" size="mini">
+                  <FormattedMessage id="Scheduled" defaultMessage="Scheduled" />
+                </Button>
+              )}
+          </Link>
+        </Table.Cell>
+        {map(indexes, (index) => (
+          <Table.Cell
+            className={cx('', { 'dragging-cell': isDragging })}
+            key={index.id}
+          >
+            {index.type === 'boolean' &&
+              (item[index.id] ? (
+                <FormattedMessage id="Yes" defaultMessage="Yes" />
+              ) : (
+                <FormattedMessage id="No" defaultMessage="No" />
+              ))}
+            {index.type === 'string' &&
+              index.id !== 'review_state' &&
+              item[index.id]}
+            {index.id === 'review_state' && (
+              <div>
+                <span>
+                  <Circle color={getColor(item[index.id])} size="15px" />
+                </span>
+                {messages[item[index.id]]
+                  ? intl.formatMessage(messages[item[index.id]])
+                  : item['review_title'] ||
+                    item['review_state'] ||
+                    intl.formatMessage(messages.no_workflow_state)}
+              </div>
+            )}
+            {index.type === 'date' && (
+              <>
+                {item[index?.id] && item[index.id] !== 'None' ? (
+                  <FormattedDate date={item[index.id]} />
+                ) : (
+                  intl.formatMessage(messages.none)
+                )}
+              </>
+            )}
+            {index.type === 'array' && (
+              <span>{item[index.id]?.join(', ')}</span>
+            )}
+            {index.type === 'object' && (
+              <span>{item[index.id]?.map((obj) => obj.title).join(', ')}</span>
```

Comment:
<span>{this.Indexes[index.id].component?this.Indexes[index.id].component({item:item}):item[index.id]?.map((obj) => obj.title).join(', ')}</span>